topics/proteomics/tutorials/proteome_annotation/tutorial.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: tutorial_hands_on
+
+title: Annotating a protein list identified by LC-MS/MS experiments
+
+zenodo_link:
+- 'https://zenodo.org/record/2650868'
+- 'https://zenodo.org/record/2650874'
+- 'https://zenodo.org/record/2650872'
+
+questions:
+- How to filter out technical contaminants?
+- How to check for tissue-specificity?
+- How to perform enrichment analysis?
+- How to map your protein list to pathways (Reactome)?
+- How to compare your proteome with other studies?
+objectives:
+- Execute a complete annotation pipeline of a protein list identified by LC-MS/MS experiments
+time_estimation: 60M
+key_points:
+
+contributors:
+- vloux
+- combesf
+- davidchristiany
+- yvandenb
+
+---
+
+# Introduction
+{:.no_toc}
+
+[ProteoRE Galaxy instance](http://www.proteore.org) provides necessary tools to execute a whole annotation pipeline of a protein list identified by LC-MS/MS experiments. This activity introduces these tools and guides you through a simple pipeline using some example datasets based on {% cite Lacombe2018 %}
+
+Once identified and/or quantified using a MS-based approach, interpreting the proteome in a sample is an important step to characterize its content in terms of functional properties in order to extend the biological knowledge related to this sample. In this activity, we illustrate the annotation and the exploration of the human exhaled breath condensate (EBC) proteome by performing the following steps:
+
+
+> ### Agenda
+>
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Get Input Datasets
+
+For this tutorial, we will use 3 datasets: the list of proteins identified by LC-MS/MS in the exhaled breath condensate (EBC) from {% cite Lacombe2018 %} and two others EBC proteomes previously published.
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial and give it a name
+>   {% include snippets/create_new_history.md %}
+>
+> 2. Import the files from [Zenodo](https://zenodo.org) or from the shared data library (ask your instructors).
+>
+>   The datasets are available on Zenodo under the references: [2650868](https://zenodo.org/record/2650868) for Lacombe et
+>   al., [2650874](https://zenodo.org/record/2650874) for Mucilli and [2650872](https://zenodo.org/record/2650872) for
```

Comment:
Please make a list of all the URLs for all files (not just the Zenodo project URL), that way users can just copy paste these into the Galaxy upload menu easily. 

For example like in this tutorial: https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html#data-upload